tests/e2e/functional_test.go

```diff
@@ -374,19 +374,18 @@ var _ = Describe("Functional e2e", Serial, func() {
 		document("verifying PipelineRollout spec deployed")
 		verifyPipelineRolloutDeployed(pipelineRolloutName)
 
-		// Give it a little while to get to Paused and then verify that it stays that way
-
+		// Give it a little while to get to Paused and then verify that it stays in Paused (or otherwise Pausing)
 		verifyPipelinePaused(Namespace, pipelineRolloutName, pipelineName)
-		// TODO: add back after Numaflow fixes this to not go from Paused to Pausing
-		//document("verifying Pipeline stays paused")
-		/*Consistently(func() bool {
+		document("verifying Pipeline stays in paused or otherwise pausing")
```

Comment:
I've noticed that it goes to "Pausing" - @kohlisid - is this understood that this may happen if we simply set `desiredPhase=Paused` and then don't make any other spec change?

Reply:
(note that this is after we confirmed it went to "Paused", it switched back to "Pausing")